scriptmodules/emulators/atari800.sh

```diff
@@ -94,6 +94,9 @@ function configure_atari800() {
     # if we are on fkms, use the sdl1 dispmanx backend by default for good performance without using X11/opengl
     isPlatform kms && isPlatform "dispmanx" && _backend_set_atari800 "dispmanx"
 
+    # when no dispmanx is available, but still on KMS, use 'sdl12-compat' and go through SDL2
+    isPlatform "kms" && ! isPlatform "dispmanx" _&& backend_set_atari800 "sdl12-compat"
```

Comment:
Should this be `_backend_set_atari800 "sdl12-compat"` ?

Reply:
Yes, I'll fix the typo.